ScaleFast.roboFontExt/lib/scaleFast.py

```diff
@@ -39,6 +39,7 @@
 from dynamicParameters.baseParameter import SingleValueParameter
 
 from vanilla import *
+from lib.UI.integerEditText import IntegerEditText
```

Comment:
could you use `from mojo.UI import NumberEditText`
```python
NumberEditText(posSize, allowFloat=False, allowNegative=True, allowEmpty=True, minimum=None, maximum=None, decimals=2, continuous=True)
```

Reply:
Done! See latest commit description for info on another thing I did.

By the way, is there a quick way to make this accept arrow keys?
https://github.com/ryanbugden/ScaleFast/blob/471ad09d769963427c7a995ed0b19550ddbe729a/ScaleFast.roboFontExt/lib/scaleFast.py#L206